### DISCUSSION TITLE
how to get battlepass info and other?

### DISCUSSION THREAD

**Discussion by gromaxbro** [LeleDerGrasshalmi/FortniteEndpointsDocumentation#250] [Q&A] [answered]
like which endpoint ?

**Comment by LeleDerGrasshalmi** [comment]
all in the athena profile

**Comment by gromaxbro** [comment]
wym

**Reply by LeleDerGrasshalmi** [reply, marked as answer]
https://github.com/LeleDerGrasshalmi/FortniteEndpointsDocumentation/tree/main/Fortnite/MCP

**Comment by gromaxbro** [comment]
and is there any api to get skins or cosmetics?

**Reply by LeleDerGrasshalmi** [reply]
everything is in the athena profile, please use the reply feature,,

**Reply by gromaxbro** [reply]
alr
